invoices/invoiceregistry.go

```diff
@@ -978,7 +984,7 @@ func (i *InvoiceRegistry) NotifyExitHopHtlc(rHash lntypes.Hash,
 // that should be executed inside the registry lock.
 func (i *InvoiceRegistry) notifyExitHopHtlcLocked(
 	ctx *invoiceUpdateCtx, hodlChan chan<- interface{}) (
-	HtlcResolution, error) {
+	HtlcResolution, invoiceExpiry, error) {
```

Comment:
Would just make a note that the `invoiceExpiry` needs to be added to the watcher.

Reply:
done